lib/index.js

```diff
@@ -2167,6 +2167,44 @@ addMethod('doto', function (f) {
 Stream.prototype.tap = Stream.prototype.doto;
 _.tap = _.doto;
 
+/**
+ * Applies a function, which must return a (possibly empty) highland stream, to each value from the source, and re-emits the
+ * source value when the function return ends.
+ *
+ * @id flatTap
+ * @section Transforms
+ * @name Stream.flatTap(f)
+ * @param {Function} f - the function to apply
+ * @api public
+ *
+ * const httpLog = H(new Promnise(res => {
```

Comment:
Use `_` instead of `H` for the Highland object.

```suggestion
 * const httpLog = _(new Promnise(res => {
```